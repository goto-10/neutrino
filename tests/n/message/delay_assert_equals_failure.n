import $assert;
import $core;
do $core:delay(fn => $assert:equals(1, 2));
