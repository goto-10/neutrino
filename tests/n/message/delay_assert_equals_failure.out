Error: --- backtrace ---
- :fail()
- :that(false)
- :equals(1, 2)
- lambda~(canonicalized)()
